orbs/pulumi.yml

```diff
@@ -71,10 +71,18 @@ commands:
         description:
             "The relative working directory to use, i.e. where your Pulumi.yaml is located."
         default: "."
+      copy:
+        type: boolean
+        description: "Copies config from an existing stack (default false)"
+        default: false
+      stack_config_copy:
+        type: string
+        description: "The stack name to copy from"
+        default: ""
     steps:
       - run:
           name: "pulumi stack init --stack << parameters.stack >> --secrets-provider << parameters.secrets_provider >>"
-          command: pulumi stack init --stack << parameters.stack >> --secrets-provider << parameters.secrets_provider >> --cwd << parameters.working_directory >>
+          command: pulumi stack init --stack << parameters.stack >> --secrets-provider << parameters.secrets_provider >> --cwd << parameters.working_directory >> <<# parameters.copy >>--copy-config-from << parameters.stack_config_copy >><</ parameters.copy >>
```

Comment:
Rather than having two parameters can't this check if stack_config_copy is != ""?

Reply:
I'm honestly not sure as I can't find any examples of anyone using the templating language (whatever it is) in this way. All of the `<<# ... >> ... <</ .. >>` constructs I can find are using booleans, which makes me suspicious as I agree that the check would make more logical sense. I also can't find a lot of examples or information on this format, though, that this orb was written in within the CircleCI docs or elsewhere. I suspect perhaps the format has changed since we started this orb, and I was trying to avoid a full refactor to make everything match. I'll test it and find out for now.

Reply:
> I'm honestly not sure as I can't find any examples of anyone using the templating language (whatever it is) in this way.

Yes I tried to find docs for this but failed to find anything :(

Reply:
Validated that it does act as expected with https://github.com/nimbinatus/circleci-pulumi-sample/commit/cea23bea12487e3ea677bcd854635bce6bb29013 and https://github.com/nimbinatus/circleci-pulumi-sample/pull/22 (with related workflow at https://app.circleci.com/pipelines/github/nimbinatus/circleci-pulumi-sample/151/workflows/e8d9cf63-f183-4228-8ede-69ce3c2bdb41/jobs/221). I'll push a commit up to change it. Thanks!